.github/workflows/NightlyBaselineProfiles.yaml

```diff
@@ -52,9 +52,8 @@ jobs:
           -Pandroid.experimental.testOptions.managedDevices.emulator.showKernelLogging=true
           -Pandroid.testoptions.manageddevices.emulator.gpu="swiftshader_indirect"
 
-      # This generates both baseline and startup profile and adds them into the generated folder
-      - name: Generate Baseline Profile
-        run: ./gradlew :app:generateReleaseBaselineProfile
-          -Pandroid.testInstrumentationRunnerArguments.androidx.benchmark.enabledRules=baselineprofile
-          -Pandroid.testoptions.manageddevices.emulator.gpu="swiftshader_indirect"
-          --stacktrace
\ No newline at end of file
+      - name: Build all build type and flavor permutations including baseline profiles
```

Comment:
Why is it necessary to build everything just to get baseline profiles for the prodRelease build?